two_same.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 D-Wave Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import dimod
+
+exactsolver = dimod.ExactSolver()
+
+Q = {(0, 0): 1, (1, 1): 1, (0, 1): -2}
```

Comment:
minor suggestion: perhaps breaking up the dictionary for readability?
```
Q = {(0, 0): 1,
     (1, 1): 1,
     (0, 1): -2}
```

Reply:
I respectfully disagree here. It seems too small to do that with, in my opinion. Is it OK to leave it as is?

Reply:
sure, that's fine. it's just a style decision.